packages/gatsby-theme-docs/src/components/seo.js

```diff
@@ -62,7 +62,7 @@ const SEO = (props) => {
   ].filter(Boolean);
   return (
     <Helmet
-      titleTemplate={`%s | ${siteData.siteMetadata.title} | commercetools`}
+      titleTemplate={`%s | ${siteData.siteMetadata.title} | ${siteData.siteMetadata.productName}`}
```

Comment:
I personally like the order here:
First the page title > then the website title > then the parent product name.